### PR TITLE
Experimtents section text replacing. Experiments section figures uplo…

### PR DESCRIPTION
**Pull request by WeihaoZhuang**
I have replaced the text in the experimental section and plotted all the images needed for the paper.

### CODE REVIEW

**Comment by TristHas** [comment]
Hi SHo-kun,

I need the following fixes from your commit:
・ lobs_lof.jpg is missing
・ In Figure 2, replace I0, I1 and I0.5 by the notation v(x,y,0), v(x,y,1)
and v(x,y,0.5)
・ In Figure 3, please add a legend for the red line saying "Interpolated
frame OF loss"
・ Figure 4 and 5 are not correctly formatted. We can not submit a paper
with bad formatting like that.
・ Both figures are too big (We are limited to 9 pages, so both figures
should fit in less than one page)
       ・ In figure 4, every figure has a different size and they are not
aligned
       ・ You need to write the captions. In particular for the quantitative
analysis, what are you showing? What are your conclusions?

Best,
Tristan



Le mer. 11 mai 2022 à 19:21, WeihaoZhuang ***@***.***> a
écrit :

**Comment by TristHas** [comment]
I forgot, Figure 6 and 7 are also too big and missing captions, please fix
it.

Le jeu. 12 mai 2022 à 14:24, Tristan Hascoet ***@***.***> a
écrit :

**Comment by TristHas** [comment]
Also, The results of the interpolated frame in Figure 3 is around 38, but
all the results of figure 4 are less than 35.
Why is that?
In Figure 3, do you compare models with the same hyperparameters (width,
depth, etc,) with and without the OF loss?


Le jeu. 12 mai 2022 à 14:34, Tristan Hascoet ***@***.***> a
écrit :

**Comment by TristHas** [comment]
Also, did you use all the frames of all the videos in all the dataset?
Or did you only use the first 40 frames or something?
If you did not use all frames of all videos, then add this explanation to
the beginning of Section4


Le jeu. 12 mai 2022 à 14:39, Tristan Hascoet ***@***.***> a
écrit :

**Comment by TristHas** [comment]
Also, are you sure you used these settings for all the ablation study:


   1.

   181  Unless specified other-wise, all experiments are run with a SIREN
   of depth 6 and width 720. We use
   2.

   182  an omega of 25 and a lambda of 0.12. We optimize the models using
   the Adam optimizer using a
   3.

   183  cosine learning rate with maximum learning rate of 3.6e-5 during
   15k epochs.


I think you only used 5000 epochs no?

Le jeu. 12 mai 2022 à 15:12, Tristan Hascoet ***@***.***> a
écrit :

**Comment by TristHas** [comment]
Also in Figure 6, you showe I0 instead of I0.5 next to the heatmap...
There are too many errors.

Le jeu. 12 mai 2022 à 15:28, Tristan Hascoet ***@***.***> a
écrit :
